packages/harmony/src/components/input/SelectablePill/SelectablePill.module.css

```diff
@@ -21,30 +21,39 @@
   box-shadow: var(--harmony-shadow-near);
 }
 
-.pill:hover {
+.pill:not(.disabled):hover {
```

Comment:
Is this generally the best css practice for handling disabled? Seems reasonable to me

Reply:
honestly got it from stackoverflow, open to ideas but also seems reasonable to me.

Reply:
IIRC, disabling pointer events would prevent hover state from being triggered?

```
.pill.disabled {
  pointer-events: none;
}
```

Reply:
nice thx randy this is better

Reply:
much better! thank you